static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -103,8 +103,7 @@ export const init = (): Promise<void> => {
         );
 
         onIabConsentNotification(state => {
-            const consentState =
-                state[1] && state[2] && state[3] && state[4] && state[5];
+            const consentState = !Object.values(state).includes(false);
```

Comment:
Maybe we can change the name of `consentState` as I don't think the variable name is helpful for someone trying to determine how this value is now used? eg. when the value of this `consentState` variable is `true` we **disable** personalised ads, which seems backwards to me. 

Maybe `turnOffPersonalisedAds` (or something similar) is a more helpful name?